test/PackageVersionsTest/VersionsTest.php

```diff
@@ -23,6 +23,10 @@ public function testValidVersions()
                 $package['version'] . '@' . $package['source']['reference'],
                 Versions::getVersion($package['name'])
             );
+            self::assertSame(
```

Comment:
Should be in a separate test method, testing only `getVersion`. Lock data fetching should be moved to a data provider